pkg/storage/mongoutil/connection.go

```diff
@@ -0,0 +1,39 @@
+package mongoutil
```

Comment:
can we do mongo/storage/conn instead of util?

Reply:
yup, absolutely